pypit/arsort.py

```diff
@@ -465,7 +475,10 @@ def match_science(fitsdict, filesort):
                 # Select the closest calibration frames to the science frame
                 tdiff = np.abs(fitsdict['time'][n].astype(np.float64)-np.float64(fitsdict['time'][iSCI[i]]))
                 wa = np.argsort(tdiff)
-                settings.spect[ftag[ft]]['index'].append(n[wa[:numfr]])
+                if settings.argflag['run']['calcheck']:
+                    settings.spect[ftag[ft]]['index'].append(n[wa])
+                else:
+                    settings.spect[ftag[ft]]['index'].append(n[wa[:numfr]])
```

Comment:
I guess I'm not sure.  For the pypit_setup script, I have set it so that 0 calib files
are required to run other than 1 arc frame.  Otherwise, the code was crashing out.
But, this was generating a 0 entry list for everything except the arcs.

I've now modified it so that one can input -1 for the non-arc files.  The setup script will 
still only return the name of a single arc frame, but I think that is ok.